docs/make.jl

```diff
@@ -15,7 +15,7 @@ using Glob
 
 GENERATED = joinpath(@__DIR__, "src", "generated")
 SOURCE = joinpath(@__DIR__, "literate")
-for subfolder ∈ ["how_to", "intro", "tutorials", "reference"]
+for subfolder ∈ ["how_to", "intro", "tutorials", "explanations"]
```

Comment:
I think reference and explanations are two distinct categories

Reference more on implementation details, explanation on explaining an concept. I agree with the change for the topolot 2d-3d change :)

Reply:
well, it both explain the concept and shows how to implement something :D

Reply:
the idea of reference is on how it is implemented, not how something should be impleented by the user.